x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/hooks/use_endpoint_action_items.tsx

```diff
@@ -125,15 +138,25 @@ export const useEndpointActionItems = (
                 'data-test-subj': 'console',
                 icon: 'console',
                 key: 'consoleLink',
+                disabled: !isResponderCapabilitiesEnabled,
                 onClick: (ev: React.MouseEvent) => {
                   ev.preventDefault();
                   showEndpointResponseActionsConsole(endpointMetadata);
                 },
-                children: (
-                  <FormattedMessage
-                    id="xpack.securitySolution.endpoint.actions.console"
-                    defaultMessage="Launch responder"
-                  />
+                children: isResponderCapabilitiesEnabled ? (
+                  launchResponderMenuText
+                ) : (
+                  <EuiToolTip
```

Comment:
You don't need to use `EuiToolTip` here because `EuiContextMenuItem` ([link](https://elastic.github.io/eui/#/navigation/context-menu)) already supports tooltips. Looks the `tooltip*` props for that component.

Reply:
gotcha thanks Paul!